packages/fluentui/react-northstar/test/__mocks__/popper.js.ts

```diff
@@ -1,14 +0,0 @@
-import * as PopperJs from 'popper.js';
```

Comment:
Consumers might be using similar mock. Don't you want to describe it in breaking changes?